### PR TITLE
Add deprecation notice

### PR DESCRIPTION
**Pull request by geddski**
### WHAT is this pull request doing?

Adds a deprecation notice to the top of the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# `@shopify/koa-shopify-auth`
+# DEPRECATED `@shopify/koa-shopify-auth`
+
+NOTE: this repo is no longer maintained. See [simple-koa-shopify-auth](https://github.com/TheSecurityDev/simple-koa-shopify-auth) for a potential community solution.
```
Comment (paulomarg) [inline README.md]: We should probably also direct folks to https://github.com/Shopify/shopify-api-node as the new official solution here!

Comment (geddski) [reply README.md]: good idea

**Review by paulomarg — APPROVED** [review]

**Pull request merged**